test.go

```diff
@@ -154,6 +183,53 @@ func (client *Client) DeleteTest(test *Test) error {
 	return client.deleteResource("test", test.ID, fmt.Sprintf("/buckets/%s/tests/%s", test.Bucket.Key, test.ID))
 }
 
+// ReadTestMetrics retrieves metrics for a test. See https://www.runscope.com/docs/api/metrics
+func (client *Client) ReadTestMetrics(test *Test, input *ReadMetricsInput) (*TestMetric, error) {
+
+	region := input.Region
+	timeframe := input.Timeframe
+	environmentUUID := input.EnvironemntUUID
+	if region == "" {
+		region = "all"
+	}
+	if timeframe == "" {
+		timeframe = "month"
```

Comment:
@sam-io out of interest why do we default to month, api default is day?

Reply:
probably because that is the value that I needed for the project I was working on, the default should be day though